utils/get_privatedns/get_privatedns.py

```diff
@@ -384,3 +405,6 @@ def main(args):
         print("\n")
         sys.stdout.flush()
         print_stderr("Keyboard interrupt, exiting...\n")
+        CURSOR.show_cursor()
```

Comment:
this one was needed even with the guy in `finally`?

Reply:
That's a good point, I added the `finally` afterwards and did not think to remove the first call to `show_cursor`